components/02-form-elements/unit/unit.config.js

```diff
@@ -0,0 +1,97 @@
+module.exports = {
+  "title": "Unit",
+  "default": "metres",
+  "collated": true,
+  "status": "ready",
+  "variants": [{
+    "name": "centimetres",
+    "context": {
+      "label": "Centimetres",
+      "unit": " cm"
+    },
+  }, {
+    "name": "metres",
+    "context": {
+      "label": "Metres",
+      "unit": " m"
+    },
+  }, {
+    "name": "kilometres",
+    "context": {
+      "label": "Kilometres",
+      "unit": " km"
+    },
+  }, {
+    "name": "miles",
+    "context": {
+      "label": "Miles",
+      "unit": " mi"
+    },
+  }, {
+    "name": "square-centimetres",
+    "context": {
+      "label": "Square-centimetres",
+      "unit": " cm²"
+    },
+  }, {
+    "name": "square-metres",
+    "context": {
+      "label": "Square-metres",
+      "unit": " m²"
+    },
+  }, {
+    "name": "square-kilometres",
+    "context": {
+      "label": "Square-kilometres",
+      "unit": " km²"
+    },
+  }, {
+    "name": "square-miles",
+    "context": {
+      "label": "Square-miles",
+      "unit": " sq mi"
```

Comment:
Could use `mi²` here to help it fit into box

Reply:
https://github.com/ONSdigital/sdc-global-design-patterns/pull/29 will amake the box bigger, as we did in survey runner.